fluent/test/fixtures_structure/elements_indent.json

```diff
@@ -7,10 +7,5 @@
     }
   },
   "baz": "Baz",
-  "qux": {
-    "val": "Qux",
-    "tags": [
-      "tag1"
-    ]
-  }
+  "qux": "Qux\n#tag1"
```

Comment:
Tags are no longer parsed by the runtime parsers so this simply becomes a multiline message value.

Reply:
I think it'd be good to unsupport tags in the tests, and probably relnotes.

Removing the tags from the ftl sources feels more discoverable of the change actually happening.

Reply:
I cherry picked ca5f6040266fe70c7adb72d3d9ab5322e99c6fb9 as the first commit in the PR.